controllers/experimentController.js

```diff
@@ -178,111 +173,111 @@ exports.experimentController = {
             await axios.get(`${growthRender}experiments/FF/${req.params.account}`)
                 .then(response => {
                     logger.info("getting FF experiments by account from Growth");
-                    res.status(200).json(response.data);
+                    res.status(200);
+                    res.json(response.data);
                 })
                 .catch(mock => {
-                    logger.info("getting FF experiments by account from mock data");
-                    const data = experimentRepository.getFeatureFlagByAccount(req.params.account);
-                    res.status(200).json(data);
+                    logger.error("Failed to get Feature Flag Experiments by account from Growth API");
+                    res.status(200);
```

Comment:
When catching the error for the API call, the response status code is set to 200, which is incorrect. It should be set to 404 to indicate that the requested resource was not found.